app/views/admin/roles/_role_manager.haml

```diff
@@ -0,0 +1,60 @@
+.card.roles.d-flex{ data: { controller: 'role-manager'}}
+  - @batch = ModelForm.new
+  = simple_form_for @batch, as: :role, url: url, method: :patch do |f|
+    .card-header
+      .d-flex
+        .user-role.w-25.d-flex.border-right.border-dark
+          %h2 User Roles
+          .ml-auto
+            = link_to path_for_new, class: 'btn btn-secondary btn-sm' do
+              %span.icon-plus
+              Add Role
+        %h2.permission-header.mr-4 Permissions
+        .form-change-count.mr-4.my-auto{ data: { 'role-manager' => { target: 'changeCount' }}}
+        .mr-auto.my-auto.hide{ data: { 'role-manager' => { target: 'changeButton' }}}= f.submit 'Save Changes', class: 'btn btn-sm'
+    .d-flex
+      .w-25
+        .card-body.border-right.border-dark
+          .administrative-permissions.d-flex.mb-4
+            .btn-group.btn-group-toggle{data: {toggle: :buttons}, aria: {label: "Toggle Administrative Permissions"}}
+              %label.btn.btn-secondary.btn-xs.active{ data: {'role-manager' => { target: 'administrativeFilter' }, action: 'click->role-manager#toggleAdmin' }}
+                %input{type: :radio, name: :roles, id: 'on-administrative-permissions', checked: :checked, value: 'show'} Show
+              %label.btn.btn-secondary.btn-xs{ data: {'role-manager' => { target: 'administrativeFilter' }, action: 'click->role-manager#toggleAdmin' }}
+                %input{type: :radio, name: :roles, id: 'off-administrative-permissions', value: 'hide'} Hide
+            .ml-4
+              Administrative Permissions
```

Comment:
Take or leave : Placement of this is a little funky to me. At first it looked like it was acting as a label for the search input field. Maybe the search can go above it? Or give the search a label (in addition to the placeholder text)?